src/components/editors.tsx

```diff
@@ -28,30 +29,17 @@ import {
 } from 'src/utils/document-model';
 import Button from './button';
 
-export interface EditorProps<
+export type EditorProps<
     T = unknown,
     A extends Action = Action,
     LocalState = unknown,
-> {
+> = UiNodes & {
     document: Document<T, A, LocalState>;
-    onChange?: (document: Document<T, A, LocalState>) => void;
```

Comment:
Tidied up and simplified these props. We were passing the same value twice for `fileId` and also `fileNodeId`